infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/schema/TableDefinition.java

```diff
@@ -117,4 +119,33 @@ public void validate() {
 	public String getCacheConfiguration() {
 		return cacheConfiguration;
 	}
+
+	public boolean isDescribedIn(FileDescriptor fileDescriptor) {
```

Comment:
All these **isDescribedIn** methods could have implemented with a visitor pattern, but I've preferred to using the simple OO delegation. WDYT @Sanne?

Reply:
The pattern that `Shall Not Be Named`! :D 

The problem with the visitor pattern is that makes it hard to follow the logic sometimes, it could  make sense in this case but, because the code is simple enough, I don't mind the approach you have used. We might refactor the code later if needed.